src/puppetlabs/puppetdb/reports.clj

```diff
@@ -153,34 +153,40 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Reports Query -> Wire format conversions
 
+(defn remove-reports-metadata [resource-events]
+  (map #(dissoc %
+                :report :certname :containing_class :configuration_version
+                :run_start_time :run_end_time :report_receive_time :environment)
+       resource-events))
+
 (pls/defn-validated resource-events-query->wire-v5 :- [resource-event-v5-wireformat-schema]
-  [events :- resource-events-expanded-query-schema]
-  (sort-by
-   #(mapv % [:timestamp :resource_type :resource_title :property])
-   (map
-    #(dissoc % :report :certname :containing_class :configuration_version
-             :run_start_time :run_end_time :report_receive_time :environment)
-    (:data events))))
-
-(pls/defn-validated logs-query->wire-v5 :- [log-wireformat-schema]
-  [logs :- logs-expanded-query-schema]
-  (:data logs))
-
-(pls/defn-validated metrics-query->wire-v5 :- [metric-wireformat-schema]
-  [metrics :- metrics-expanded-query-schema]
-  (:data metrics))
+  [resource-events :- resource-events-expanded-query-schema]
+  (->> resource-events
+       :data
+       remove-reports-metadata
+       (sort-by #(mapv % [:timestamp :resource_type :resource_title :property]))))
```

Comment:
Oh I validated it wasn't needed in anonymizer, I'll remove it here as well since we don't need it here either